pyonfx/shape.py

```diff
@@ -845,12 +850,12 @@ def heart(size: float, offset: float = 0) -> Shape:
         # Build shape from template
         shape = Shape(
             "m 15 30 b 27 22 30 18 30 14 30 8 22 0 15 10 8 0 0 8 0 14 0 18 3 22 15 30"
-        ).map(lambda x, y: (x * mult, y * mult))
+        ).map(lambda x, y, typ: (x * mult, y * mult))
 
         # Shift mid point of heart vertically
         count = 0
 
-        def shift_mid_point(x, y):
+        def shift_mid_point(x: float, y: float, _: Any) -> Tuple[float, float]:
```

Comment:
For internal functions we decided to not add type hinting, as some internal functions could make the code really verbose, and final users do not benefit from this.